modules/social_features/social_event/social_event.services.yml

```diff
@@ -19,3 +19,8 @@ services:
     class: Drupal\social_event\Service\SocialEventEnrollService
     arguments:
       - '@config.factory'
+    deprecated: The "%service_id%" service is deprecated in social:11.4.0 and is removed from social:12.0.0. See https://www.drupal.org/project/social/issues/3306568
```

Comment:
@nechai I think we need to created separate Change Record for this deprecation.
Also it should go to 11.5.0 and 12.0.0 then.